tests/c01 - GendtseWaardNevengeul/Qmin_4000_v2_rkm.cfg

```diff
@@ -14,14 +14,14 @@
 [C1]
   Discharge   = 3000.0
   Reference   = reference-Q1_map.nc
-  WithMeasure = reference-Q1_map.nc
+  WithIntervention = reference-Q1_map.nc
 
 [C2]
   Discharge   = 4000.0
   Reference   = reference-Q2_map.nc
-  WithMeasure = reference-Q2_map.nc
+  WithIntervention = reference-Q2_map.nc
 
 [C3]
   Discharge   = 6000.0
   Reference   = reference-Q3_map.nc
-  WithMeasure = measure-Q3_map.nc
+  WithIntervention = measure-Q3_map.nc
```

Comment:
is this the correct file name? in docs/chapters/testing.tex we refer to a different name

Reply:
lets have a look together

Reply:
I found on line 1305 in backwards_compatibility.tex a config, not sure if it is related to this.
I found on line 222 in file_formats.tex a config, not sure if it is related to this.
This is updated to intervention-Q3_map.nc, so expect to be correct now.